easybuild/easyconfigs/i/IOHexperimenter/IOHexperimenter-0.2.9.2-foss-2020b.eb

```
@@ -0,0 +1,33 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonPackage'
+
+name = 'IOHexperimenter'
+version = '0.2.9.2'
+
+homepage = "https://pypi.org/project/IOHexperimenter/"
+description = """this is the python interface to iohexperimenter, which is the experimentation environment for 
+iterative optimization heuristics (iohs). the source code of iohexperimenter is written in c++ and its 
+python interface is realized using swig. iohexperimenter is a part of the bigger project called iohprofiler."""
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
+]
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),  # provides numexpr
```

```suggestion
    ('SciPy-bundle', '2020.11'),
```
I think the comment has been copied from another easyconfig. It's probably not relevant here.